### PR TITLE
ansible.network job template added

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

### CODE REVIEW

File: zuul.d/project-templates.yaml [hunk 1]
```diff
@@ -775,6 +775,33 @@
       queue: integrated
       jobs: *ansible-collections-ansible-netcommon-units-jobs
 
+- project-template:
+    name: ansible-collections-ansible-network-units
+    check:
+      jobs: &ansible-collections-ansible-network-units-jobs
+        - ansible-changelog-fragment
+        - ansible-test-units-netcommon-python27
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:783]: lets drop all units testing, we don't have any python code in repo

File: zuul.d/project-templates.yaml [hunk 2]
```diff
@@ -775,6 +775,33 @@
...
+        - ansible-test-units-netcommon-python35
+        - ansible-test-units-netcommon-python36
+        - ansible-test-units-netcommon-python37
+        - ansible-test-units-netcommon-python38
+
+        - build-ansible-collection:
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:789]: when you respin, can you sort this list.

We also need vyos.vyos

Comment (rohitthakur2590) [reply zuul.d/project-templates.yaml:784]: updated!

File: zuul.d/project-templates.yaml [hunk 3]
```diff
@@ -775,6 +775,29 @@
...
 
+- project-template:
+    name: ansible-collections-ansible-network-units
+    check:
+      jobs: &ansible-collections-ansible-network-units-jobs
+        - ansible-changelog-fragment
+
+        - build-ansible-collection:
+            required-projects:
+              - name: github.com/ansible-collections/ansible.netcommon
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:786]: doh, this should also have ansible-collection/ansible.network too.  Sorry

File: zuul.d/project-templates.yaml [hunk 4]
```diff
@@ -775,6 +775,29 @@
       queue: integrated
       jobs: *ansible-collections-ansible-netcommon-units-jobs
 
+- project-template:
+    name: ansible-collections-ansible-network-units
+    check:
+      jobs: &ansible-collections-ansible-network-units-jobs
+        - ansible-changelog-fragment
+
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:783]: lets remove white space

File: zuul.d/project-templates.yaml (hunk 3 above)
Comment (rohitthakur2590) [reply zuul.d/project-templates.yaml:785]: updated. Thanks

File: zuul.d/project-templates.yaml (hunk 4 above)
Comment (rohitthakur2590) [reply zuul.d/project-templates.yaml]: Done. Thanks

**Pull request merged**